antora-playbook.yml

```diff
@@ -18,7 +18,8 @@ content:
       branches:
         - master_asciidoctor
     - url: https://github.com/bonitasoft/bonita-doc.git
-      branches:
+      -branches:
+        - old_versions
```

Comment:
Thx, I didn't noticed /=
It's because of the Antora default way to sort versions, textual versions have a higher priority than classic versions (using semantic versioning). 
I had to create a custom sorter to display versions is the correct order (see attached pr on theme repo), but I forgot about the default selected version. 
The solution is here: https://docs.antora.org/antora/2.3/playbook/site-start-page/#use-a-pages-latest-version

I have an other solution in mind: instead of using "old" as the version for the legacy branch, we could use a low version like 7.2 or whatever bellow. It would work like a charm naturally, without any custom sorter or additional conf. The only issue is that the url of the legacy page will contain this version, which doesn't have any real meaning.

What do you think?